recipes/r-ffield/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = '0.1.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ffield
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: FField_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/FField_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/FField/FField_{{ version }}.tar.gz
+  sha256: 18c5fbcf3f7c9151ee528df3be1156febf5d3f6fc650e1ec829898bdc5360117
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-ggplot2
+    - r-gridextra
+  run:
+    - r-base
+    - r-ggplot2
+    - r-gridextra
+
+test:
+  commands:
+    - $R -e "library('FField')"  # [not win]
+    - "\"%R%\" -e \"library('FField')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=FField
+  license: GPL-3
```

Comment:
can you please add a license_file as well like here: https://github.com/bgruening/conda_r_skeleton_helper/blob/master/run.sh#L20

Reply:
Done, thanks for the hint!